src/transport/SecureSession.cpp

```diff
@@ -132,11 +133,32 @@ CHIP_ERROR SecureSession::GetIV(const MessageHeader & header, uint8_t * iv, size
     return err;
 }
 
+CHIP_ERROR SecureSession::GetAAD(const MessageHeader & header, uint8_t * aad, size_t & len)
```

Comment:
This method seems to be just:

header.Encode(aad, len, &len);

do we plan to add more functionality (if so, should we add some TODO)?

Reply:
No other functionality will be added. I moved it to a function to be more descriptive. But as you suggested in a latter comment, the name is not descriptive enough. I'll fix that and add a comment too.

Reply:
Added approval, however I am still unsure if an extra level of indirection really helps readability.